content/docs/cve/cveTeams.md

```diff
@@ -0,0 +1,55 @@
+---
+# Title, summary, and page position.
+linktitle: "Teams: Session Management"
```

Comment:
```suggestion
linktitle: Create and Edit Teams in ExamTools
```

Sorry, i just noticed that the title you use here is different from the title you have elsewhere. For consistent style I think it should be like this.

<img width="934" alt="Screen Shot 2022-02-16 at 1 39 16 PM" src="https://user-images.githubusercontent.com/4356471/154333733-8b7e2168-89ac-4f81-b313-7fa64b6a56a2.png">